src/deprecated.jl

```diff
@@ -1,6 +1,6 @@
 @deprecate Progress(n::Integer, dt::Real, desc::AbstractString="Progress: ",
-    barlen=nothing, color::Symbol=:green, output::IO=stderr;
-    offset::Integer=0) Progress(n; dt=dt, desc=desc, barlen=barlen, color=color, output=output, offset=offset)
+    barlen=nothing, barlen_fraction::Float64=1.0, color::Symbol=:green, output::IO=stderr;
```

Comment:
don't add it to the deprecated signature, we don't want to use these anymore. anyway it wouldn't be compatible with previous versions because it's not at the end